src/serlo-editor/plugins/serlo-template-plugins/exercise-group/text-exercise-group.tsx

```diff
@@ -76,6 +78,7 @@ function TextExerciseGroupTypeEditor(
   return (
     <>
       <div className="absolute right-0 -mt-20 mr-side flex">
+        {canUseAiFeaturesOutsideProduction && <AiExerciseGenerationButton />}
```

Comment:
works fine here but I personally stopped using `&&` in jsx in general to avoid bugs.
(reasoning e.g. [this article](https://www.crocoder.dev/blog/react-conditional-rendering/))